security-admin/scripts/setup.sh

```diff
@@ -673,7 +676,28 @@ update_properties() {
 	newPropertyValue="${policymgr_supportedcomponents}"
 	updatePropertyToFilePy $propertyName $newPropertyValue $to_file_ranger
 
-	propertyName=ranger.jpa.jdbc.user
+	if [ "${policymgr_user_group_management_enabled}" != "" ]
+	then
+        propertyName=ranger.user.group.management.enabled
```

Comment:
Indentation is a off for this and subsequent blocks. Please address.

Reply:
This looks strange. In my editor, the blocks look fine. I will check this issue. Thanks for pointing out.

![screen shot 2017-11-08 at 5 44 06 pm](https://user-images.githubusercontent.com/32578398/32584347-ab9ea740-c4ac-11e7-8186-b0c474d3750b.png)